interface/src/app/gene/network/network.js

```diff
@@ -175,57 +193,52 @@ angular.module('adage.gene.network', [
         }
 
         /**
-         * Callback function to show edge tips when mouse is over an edge.
+         * Callback function to show gene tips when a gene is clicked.
+         * @param {gene_data} data;
+         * @return {void}.
+         */
+        function showGeneTip(data) {
+          edgeTip.hide(); // Hide edge-tip window (if any) first.
+          geneTip.show(data);
+        }
+
+        /**
+         * Callback function to show edge tips when an edge is clicked.
          * @param {edge_data} data;
          * @return {void}.
          */
         function showEdgeTip(data) {
+          geneTip.hide(); // Hide gene-tip window (if any) first.
           var rawWeight = data.weight + rawMinWeight;
-          var str = 'Edge weight: ' + rawWeight.toFixed(3);
+          var htmlText = 'Edge weight: ' + rawWeight.toFixed(3);
```

Comment:
Nice, I like this more descriptive variable name :+1: . Should the 3 be defined somewhere above this as a setting/parameter rather than a magic number of how many decimal digits we want?